OpenRA.Mods.RA/Scripting/Properties/InfiltrateProperties.cs

```diff
@@ -19,18 +22,23 @@ namespace OpenRA.Mods.RA.Scripting
 	[ScriptPropertyGroup("Ability")]
 	public class InfiltrateProperties : ScriptActorProperties, Requires<InfiltratesInfo>
 	{
-		readonly InfiltratesInfo info;
+		readonly IEnumerable<Infiltrates> infiltratesTraits;
 
 		public InfiltrateProperties(ScriptContext context, Actor self)
 			: base(context, self)
 		{
-			info = Self.Info.TraitInfo<InfiltratesInfo>();
+			infiltratesTraits = Self.TraitsImplementing<Infiltrates>();
```

Comment:
In cases like this an array is preferrable because we (in principle) enumerate the list many mroe times than we initialize it, and the improved enumeration perf beats the worse setup cost.

Of course this will almost never have more than one element, so there's not a lot of point arguing over perf.